audio/public/css/audio.css

```diff
@@ -1,6 +1,7 @@
 .audio-xblock {
   display: block;
   margin-bottom: 4rem;
+  margin-top: 4rem;
```

Comment:
This is a rather hacky way to ensure the caption selector isn't cut off at the top... not sure if there's a better way.